cli/command/registry/login.go

```diff
@@ -47,6 +51,27 @@ func runLogin(dockerCli command.Cli, opts loginOptions) error {
 	ctx := context.Background()
 	clnt := dockerCli.Client()
 
+	if opts.password != "" {
+		fmt.Fprintln(dockerCli.Err(), "WARNING! Using --password via the CLI is insecure. Use --password-stdin")
```

Comment:
Should there be a period at the end of this? If `Using --password via the CLI is insecure` is formatted as a sentence, I think `Use --password-stdin` should be as well.